nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -155,6 +179,25 @@ def _set_backend_entity(self, model: TModel) -> None:
             raise RuntimeError('Cannot return backend-specific entity'
                                'because {} is not supported!'.format(model_backend))
 
+    def _get_default_statistics_collector(self, is_symmetric: bool,
+                                          axes: Optional[Tuple], per_channel: bool) -> TensorStatisticCollectorBase:
+        """
+        Returns the default StatisticCollector.
+
+        :param is_symmetric: True if the quantizer has symmetric mode. False if asymmetric.
+        :param axes: Axes to reduce in the statistic tensor.
+        :param per_channel: True if per-channel statistics. False if per-tensor.
+        :return: StatisticCollector.
+        """
+        if per_channel:
```

Comment:
Why is the `minmax_statistic_collector` used for the per_channel? Where did this condition come from?

Reply:
This is the default statistics collector for per-channel case, based on the default statistics collector from POT